components/table/test/table-test-sticky-visual-diff.js

```diff
@@ -477,6 +477,108 @@ class TestTableStickyVisualDiff extends LitElement {
 					</table>
 				</d2l-table-wrapper>
 			</div>
+			<div class="d2l-visual-diff">
```

Comment:
This test simulates the modifications grades has had to do to make dropdowns within the row header appear "on top" of the second header row:

<img width="500" alt="Screenshot 2023-05-18 at 9 25 08 AM" src="https://github.com/BrightspaceUI/core/assets/5491151/eacfcf12-4ef8-4ce8-8ed7-f8f87a1efce4">

Note that it's a bit glitchy -- that's expected and is what #3635 will resolve. This sets the golden baseline.